.github/workflows/test.yml

```diff
@@ -150,15 +197,18 @@ jobs:
       - name: Checkout registry
         uses: ./.github/actions/checkout-registry
 
+      - name: Fetch PR base
+        run: git fetch origin $PR_BASE
+
       - name: Balance Tests
         run: |
-          if git diff $PR_BASE...HEAD --name-only | grep -qE '^typescript/infra/config/environments/mainnet3/balances|^.registryrc$'; then
+          if git diff origin/$PR_BASE...HEAD --name-only | grep -qE '^typescript/infra/config/environments/mainnet3/balances|^.registryrc$'; then
             yarn --cwd typescript/infra test:balance
           fi
 
       - name: Warp Config Tests
         run: |
-          if git diff $PR_BASE...HEAD --name-only | grep -qE '^typescript/infra/|^.registryrc$'; then
+          if git diff origin/$PR_BASE...HEAD --name-only | grep -qE '^typescript/infra/|^.registryrc$'; then
             yarn --cwd typescript/infra test:warp
           fi
```

Comment:
drive-by fix to _actually_ check the diff <--- currently the diff call errors but still skips the test lol